### DISCUSSION TITLE
How to connect to ista domaqua radio net 3

### DISCUSSION THREAD

**Discussion by LPiotr** [Ludy87/ecotrend-ista#94] [Q&A]
Hi I trying to read some data from mentioned above water meter. Do you know something about this? I am looking on entire Internet and nothing found. I am currently using DVB-T stick with RTL2838U.

**Comment by Ludy87** [comment]
Hello, the integration is about the accounts provided by Ista.
You should consult relevant forums, discords or FB groups.
